cmd/snap-update-ns/main_test.go

```diff
@@ -288,8 +285,10 @@ func (s *mainSuite) TestApplyUserFstab(c *C) {
 	err = update.ApplyUserFstab("foo")
 	c.Assert(err, IsNil)
 
+	xdgRuntimeDir := fmt.Sprintf("%s/%d", dirs.XdgRuntimeDirBase, os.Getuid())
```

Comment:
Maybe allow mocking osGetuid for testing? It could be more flexible for testing.